src/main/resources/templates/index.html

```diff
@@ -13,16 +13,16 @@
 
     <ul class="nav">
         <li><a href="index.html">Search</a></li>
-        <li><a href="AddFacility.html">Add Faculty</a></li>
+        <li><a href="addFacility.html">Add Faculty</a></li>
     </ul>
 
-    <form action="#" th:action="${/facility}" method="GET">
+    <form action="#" th:action="@{/facility}" method="GET">
     <div class="box">
         <p>Search for a recycling facility center near you!</p>
 
 
         <label for="recycleMaterial"></label>
-        <input type="text" id="recycleMaterial" name="materialName" placeholder="Recyclable Material" th:field="*{materialName}">
+        <input type="text" id="recycleMaterial" name="materialName" placeholder="Recyclable Material" th:field="*{recyclableMaterial.materialName}">
```

Comment:
Since you did not pass an attribute name with the model, you need to specify the object name followed by the attribute. This prevents the index page from loading and throws the exception below.

Error during execution of processor 'org.thymeleaf.spring5.processor.SpringInputGeneralFieldTagProcessor' (template: "index" - line 25, col 103)] with root cause

java.lang.IllegalStateException: Neither BindingResult nor plain target object for bean name 'materialName' available as a request attribute

I recommend changing this so your index page will load and not throw an exception.